config.json

```diff
@@ -89,6 +93,9 @@
     },
     "IT": {
       "government": "https://www.salute.gov.it/nuovocoronavirus"
+    },
+    "BR": {
+      "government": "https://coronavirus.saude.gov.br/"
```

Comment:
I believe the brazil geometry is not loaded in the database, so please don't include it.

Reply:
Maybe solution would be to add Brazil in DB instead ? @RockyTV are you planning to offer "Ça reste ouvert" in Brazil ? If so, please create an issue to do so : https://github.com/osmontrouge/caresteouvert/issues/new?assignees=&labels=i18n&template=expand-data-to-my-country.md&title=

Reply:
Do I need to add the country to the DB so POIs in Brazil will be displayed in the map? Also, is a custom domain name required or optional?

Reply:
All details are in given link, opening a country requires several things. Putting country limit in DB can be done by our team if other requirements are met.

Reply:
I'll edit the file once I get a response from some members of the local community to decide if we want to offer the service in Brazil. For now it's just going to be the blog post.